Project not compiling

**Describe the bug**
On github action for release the following message is failling the compilation:
Error: [typecheck:node] src/main/libraries/logic/Launchers.ts(3,1): error TS6133: 'shell' is declared but its value is never read.

**To Reproduce**
Trigger the action

**Expected behavior**
Compilation success and release

**Version**
Any

**Log file**
You can find it in %userdir%\SaveDataSynchronizer\logs\application.log
> SaveDataSynchronizer@1.3.4 build:win /home/runner/work/SaveDataSynchronizer/SaveDataSynchronizer
> pnpm build && electron-builder --win --publish never


> SaveDataSynchronizer@1.3.4 build /home/runner/work/SaveDataSynchronizer/SaveDataSynchronizer
> pnpm lint && pnpm format && pnpm typecheck && electron-vite build


> SaveDataSynchronizer@1.3.4 lint /home/runner/work/SaveDataSynchronizer/SaveDataSynchronizer
> eslint . --ext .js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix


/home/runner/work/SaveDataSynchronizer/SaveDataSynchronizer/src/main/libraries/logic/Launchers.ts
Warning:   3:[10](https://github.com/Emiliopg91/SaveDataSynchronizer/actions/runs/9820379047/job/27114882663#step:10:11)  warning  'shell' is defined but never used  @typescript-eslint/no-unused-vars

✖ 1 problem (0 errors, 1 warning)


> SaveDataSynchronizer@1.3.4 format /home/runner/work/SaveDataSynchronizer/SaveDataSynchronizer
> prettier --write .

.eslintrc.cjs 43ms (unchanged)
.github/ISSUE_TEMPLATE/bug_report.md 47ms (unchanged)
.github/workflows/main.yml 17ms (unchanged)
.prettierrc.yaml 2ms (unchanged)
.vscode/extensions.json 22ms (unchanged)
.vscode/launch.json 6ms (unchanged)
.vscode/settings.json 2ms (unchanged)
CODE_OF_CONDUCT.md 21ms (unchanged)
dev-app-update.yml 1ms (unchanged)
electron-builder.yml 6ms (unchanged)
electron.vite.config.ts 51ms (unchanged)
package-lock.json 134ms (unchanged)
package.json 26ms (unchanged)
README.md 5ms (unchanged)
src/main/applicationLogic.ts 21ms (unchanged)
src/main/index.ts [11](https://github.com/Emiliopg91/SaveDataSynchronizer/actions/runs/9820379047/job/27114882663#step:10:12)1ms (unchanged)
src/main/libraries/dtos/Configuration.ts 5ms (unchanged)
src/main/libraries/dtos/Game.ts 4ms (unchanged)
src/main/libraries/exceptions/RCloneException.ts 5ms (unchanged)
src/main/libraries/helpers/Constants.ts 7ms (unchanged)
src/main/libraries/helpers/FileUtils.ts 75ms (unchanged)
src/main/libraries/helpers/GameHelper.ts 45ms (unchanged)

--- a/src/main/libraries/logic/Launchers.ts
+++ b/src/main/libraries/logic/Launchers.ts
@@ -1,6 +1,5 @@
 import { File, LoggerMain, OSHelper } from '@tser-framework/main';
 import { spawn } from 'child_process';
-import { shell } from 'electron/common';
 import path from 'path';
 
 import { Game } from '../dtos/Game';

